common/TraceEvent.cpp

```diff
@@ -9,12 +9,12 @@
 // clang++ -Wall -Wextra -DTEST_TRACEEVENT_EXE TraceEvent.cpp -o TraceEvent -pthread
 
 #include "config.h"
```

Comment:
Please leave a blank line after `config.h`. Also, best to make it `#include <config.h>` to be consistent.